### ISSUE TITLE
Fnf Week 7 Android 

### ISSUE THREAD

**Issue by gunkilock** [luckydog7/Funkin-android#806]
HERE'S YOUR WEEK 7 FOR ANDROID NOW LEAVE EVERYONE ALONE AND DON'T COMPLAIN ABOUT WEEK 7 ON ANDROID AAAAAAAAAAAAAA
https://drive.google.com/file/d/1ArmdjXMj_0kts9ts3zblko16lsZsnJJ_/view

**Comment by EricizionS** [comment]
cant download, too many people

**Comment by gunkilock** [comment]
Bruh Just click the link and you're done week 7 downloading

**Comment by ROBIKALIVE13** [comment]
> Bruh Just click the link and you're done week 7 downloading

hes sayin the truth

**Comment by EricizionS** [comment]
i did few minutes ago, it says that theres too many people downloading

**Comment by EricizionS** [comment]
let me try again

**Comment by GlitzHere** [comment]
Is it safe?

**Comment by gunkilock** [comment]
Yes

**Comment by GlitzHere** [comment]
Thx!!!

**Comment by gunkilock** [comment]
No Problem

**Comment by Ruv023** [comment]
that moment when all the mod dialouges arnt english ;-;

**Comment by MistyFisty** [comment]
yo dudes i can download it and it will scan if it has a virus or not

**Comment by gunkilock** [comment]
No virus
